client/src/elements/atoms/button-tertiary.css

```diff
@@ -4,13 +4,8 @@ button-tertiary {
 }
 
 button-tertiary icon-button button {
-  background-color: transparent;
-  border-color: transparent;
-}
-
-button-tertiary icon-button button[disabled] {
-  color: var(--q-color-gray-5);
   border-color: transparent;
+  background-color: var(--q-color-white);
```

Comment:
Why does the button-tertiary needs a white background?- It results in the following:

<img width="174" alt="bildschirmfoto 2018-07-03 um 13 00 24" src="https://user-images.githubusercontent.com/1810384/42216174-3c628192-7ec1-11e8-8d04-b61649889c9a.png">

Reply:
it's a mistake. It needs to overwrite the :active, :focus, :hover styles in case the button is disabled. fixed now.